### PR TITLE
Update CLI install scripts

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @pakrym

File: build/KoreBuild.ps1 [hunk 1]
```diff
@@ -11,7 +11,7 @@ $koreBuildFolder = $PSScriptRoot
 $koreBuildFolder = $koreBuildFolder.Replace($repoFolder, "").TrimStart("\")
 
 $dotnetVersionFile = $koreBuildFolder + "\cli.version.win"
-$dotnetChannel = "beta"
+$dotnetChannel = "preview"
```
Comment (pranavkm) [inline build/KoreBuild.ps1:14]: The install script defaults to beta now

**Comment by pakrym** [comment]
:shipit:

**Comment by natemcmaster** [comment]
2702 isn't on the "preview" channel. It's still beta. Until we get a higher build of CLI, this will fail.

**Comment by pranavkm** [comment]
@natemcmaster - yeah. The CoreFx build is almost through (another 30 mins) and should solve this issue.

**Comment by pranavkm** [comment]
Should be fixed now - https://github.com/aspnet/KoreBuild/commit/ffe7537913d649e46e8870601dc64e2337277cba updated it to a preview2 build.

**Pull request merged**